frontend/src/locales/en-US.json

```
@@ -845,7 +845,11 @@
       "no-earlier-than": "Restore point-in-time cannot be earlier than [{earliest}] (the earliest available backup).",
       "no-later-than-now": "Restore point-in-time cannot be later than now.",
       "restore-before-last-migration": "Restore to the point before last migration",
-      "restore-before-last-migration-help-info": "@:{'database.pitr.restore-before-last-migration'}. {link}."
+      "restore-before-last-migration-help-info": "@:{'database.pitr.restore-before-last-migration'}. {link}.",
+      "will-overwrite-current-database": "Will overwrite current database",
+      "restore-to": "Restore to",
+      "restore-to-new-db": "New database",
+      "restore-to-in-place": "In place"
```

```suggestion
      "restore-to": "Restore",
      "restore-to-new-db": " To new database",
      "restore-to-in-place": "In place"
```
nit: how about this?

---

SGTM